.github/ISSUE_TEMPLATE.md

```diff
@@ -13,11 +13,12 @@ learn how: http://kcd.im/pull-request
 
 - `gatsby-plugin-pinterest` version:
 - `node` version:
-- `npm` (or `yarn`) version:
+- `npm` version:
```

Comment:
Why not yarn?

Reply:
Or is this simply automated from kcd-scripts?

Reply:
I made it equal to `kcd-scripts`